src/status_im/ui/screens/events.cljs

```diff
@@ -71,6 +72,11 @@
  (fn [coeffects _]
    (assoc coeffects :now (time/now-ms))))
 
+(re-frame/reg-cofx
```

Comment:
since we already have `now` and it is a coeffect added to all event, I think it is better to just have an `to-s` function that turns `now` into now-s rather than a second coeffect that also produces a timestamp

Reply:
yeah thought also about this